src/query/interpreter.cpp

```diff
@@ -786,8 +785,9 @@ Callback HandleReplicationQuery(ReplicationQuery *repl_query, const Parameters &
       callback.header = {
           "name", "socket_address", "sync_mode", "current_timestamp_of_replica", "number_of_timestamp_behind_master",
           "state"};
-      callback.fn = [handler = ReplQueryHandler{dbms_handler}, replica_nfields = callback.header.size()] {
-        const auto &replicas = handler.ShowReplicas();
+      callback.fn = [handler = ReplQueryHandler{dbms_handler}, replica_nfields = callback.header.size(),
+                     db_acc = current_db.db_acc_] {
+        const auto &replicas = handler.ShowReplicas(*db_acc->get());
```

Comment:
TODO: `SHOW REPLICAS;` should become DB agnostic. New query `SHOW REPLICAS STATUS;` would show the status of the replicas for the current DB.

Reply:
SHOW REPLICA refactor will be done in another PR.